### PR TITLE
[Term Entry] C++ vector .size()

### LINKED ISSUE #4981: [Term Entry] C++ vector .size()
### Reason/inspiration (optional)

"We would like a new entry on the '.size()' term entry in Cpp under `Vector` concept. The entry should go in a new file under `content/cpp/concepts/vectors/terms/size/size.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept.
- An `Example` section that provides an example demonstrating the concept in use.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by eddiepeters00**
### Description
Added a new term entry for the `.size()` method on vectors in c++.

### Issue Solved
Closes #4981

### Type of Change
- Adding a new entry

### Checklist
- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by SaviDahegaonkar — APPROVED** [review]
Hey Author,
I had a look at your PR and hereby approve it for the next review.

Thanks,
Savi

File: content/cpp/concepts/vectors/terms/size/size.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - 'Vectors'
+  - 'Arrays'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`.size()`** method returns the number of elements in the vector.
+`.size()` is there to follow the consistency of other standard library containers (e.g., [maps](https://www.codecademy.com/resources/docs/cpp/maps) and [strings](https://www.codecademy.com/resources/docs/cpp/strings)).
```
Comment (mamtawardhani) [inline content/cpp/concepts/vectors/terms/size/size.md:17]:
```suggestion
The **`.size()`** method returns the number of elements in the vector. It follows the consistency of other standard library containers, such as [maps](https://www.codecademy.com/resources/docs/cpp/maps) and [strings](https://www.codecademy.com/resources/docs/cpp/strings).
```

Comment (mamtawardhani) [reply content/cpp/concepts/vectors/terms/size/size.md:17]: Reframed for better clarity

File: content/cpp/concepts/vectors/terms/size/size.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Syntax
+
+```pseudo
+vector.size();
+```
+
+The `vector` must be defined with `std::vector` prior to using with the `.size()` method.
+
+## Codebyte Example
```
Comment (mamtawardhani) [inline content/cpp/concepts/vectors/terms/size/size.md:27]:
```suggestion
## Example
```

File: content/cpp/concepts/vectors/terms/size/size.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+In the example below, `.size()` is called on the `numbers` vector:
+
+```codebyte/cpp
```
Comment (mamtawardhani) [inline content/cpp/concepts/vectors/terms/size/size.md:31]:
```suggestion
```cpp
```

File: content/cpp/concepts/vectors/terms/size/size.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#include <vector>
+
+int main() {
+    // Declaring a vector with 4 integers
+    std::vector<int> numbers = {1, 2, 3, 4};
+
+    // Print out vector size
+    std::cout << numbers.size();
+}
+```
```
Comment (mamtawardhani) [inline content/cpp/concepts/vectors/terms/size/size.md:42]: Please add an output of the above code block below as follows: (use the ```shell``` block)

The output of the above code is:

```shell
4
```

File: content/cpp/concepts/vectors/terms/size/size.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+The **`.size()`** method returns the number of elements in the vector.
+`.size()` is there to follow the consistency of other standard library containers (e.g., [maps](https://www.codecademy.com/resources/docs/cpp/maps) and [strings](https://www.codecademy.com/resources/docs/cpp/strings)).
+
+## Syntax
+
+```pseudo
+vector.size();
+```
+
+The `vector` must be defined with `std::vector` prior to using with the `.size()` method.
```
Comment (mamtawardhani) [inline content/cpp/concepts/vectors/terms/size/size.md:25]:
```suggestion
The `vector` must be defined using `std::vector` before the `.size()` method can be used.
```

File: content/cpp/concepts/vectors/terms/size/size.md (hunk 2 above)
Comment (mamtawardhani) [reply content/cpp/concepts/vectors/terms/size/size.md:27]: For this issue, an Example block is needed

File: content/cpp/concepts/vectors/terms/size/size.md (hunk 3 above)
Comment (mamtawardhani) [reply content/cpp/concepts/vectors/terms/size/size.md:31]: In the example block, we use the ```cpp block. If it was a codebyte block, then we use the ```codebyte/cpp

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @eddiepeters00, thank you so much for your contribution to Codecademy Docs! The entry is nicely written 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? 
Also, please remove the `extensions.json` file from this branch.
Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @eddiepeters00. The entry is ready to be merged! 😄

**Pull request merged**